Documentation/tutorial/intro_toc.md

```diff
@@ -27,16 +29,25 @@ Basic format
 # @UidForAnArticle
 ```
 
-For a *TOC Item*, it can be either plain text, or a [Markdown inline link](http://daringfireball.net/projects/markdown/syntax#link), or `@uid` as the shortcut for [cross-referenced](../spec/docfx_flavored_markdown.md#cross-reference)
+> [!NOTE]
+> For a *TOC Item*, it can be either plain text, or a [Markdown inline link](http://daringfireball.net/projects/markdown/syntax#link), or `@uid` as the shortcut for [cross-reference](../spec/docfx_flavored_markdown.md#cross-reference).
 
 Three kinds of links are supported:
 
 1. Absolute path, for example, `http://example.net`.
 2. Relative path, for example, `../example.md`. This kind of link has several advanced usages and is described in detail [below](#href-in-detail).
-3. URI with `xref` scheme, for example, `xref:System.String`, the value is the `uid` of the file to be [cross-referenced](../spec/docfx_flavored_markdown.md#cross-reference).
+3. URI with `xref` scheme or `@` syntax, for example, `xref:System.String` or `@System.String`. The value is the `uid` of the file to [cross-reference](../spec/docfx_flavored_markdown.md#cross-reference).
 
 ### YAML format TOC `toc.yml`
 
+Comparing to `toc.md`, `toc.yml` represents a structured data model and conforms to the [YAML standard](http://www.yaml.org/spec/1.2/spec.html). It supports advanced functionalities.
+
+#### Data model for `toc.yml`
+
+The data model for `toc.yml` is either an **array** of *TOC Item Object*s, or an **object** consisting of two properties - "metadata": an **object** containing all metadata of the TOC, and "items": an **array** of *TOC Item Object*s.
```

Comment:
>metadata [](start = 124, length = 8)

metadata is an extensible member in toc.yml, so no need for a key named "metadata"
[https://github.com/dotnet/docfx/blob/dev/src/Microsoft.DocAsCode.DataContracts.Common/TocRootViewModel.cs#L23](https://github.com/dotnet/docfx/blob/dev/src/Microsoft.DocAsCode.DataContracts.Common/TocRootViewModel.cs#L23)

Reply:
I want to follow the same way as OPS

---
In reply to: [208489941](https://github.com/dotnet/docfx/pull/3165#discussion_r208489941) [](ancestors = 208489941)

Reply:
We can also remove this part, as users will not use it for default template.